common/src/main/java/org/jboss/hal/testsuite/page/runtime/WebServiceEndpointsPage.java

```diff
@@ -50,4 +54,14 @@ public void navigate() {
                 .invoke(FinderNames.VIEW);
         Application.waitUntilVisible();
     }
+
+    public TreeNavigation getTreeNavigation() {
+        return Graphene.createPageFragment(TreeNavigation.class, getContentRoot());
+    }
+
+    public FormItemTableFragment getFromItemTable() {
+        return Graphene.createPageFragment(FormItemTableFragment.class,
+                getContentRoot().findElement(ByJQuery.selector("table.form-item-table")));
+    }
```

Comment:
Well, `WebServiceEndpointsPage` is already extending `MetricsPage`. AFAIK not every `MetricsPage` is equipped with tree navigation. There are several other solutions available:

1. Let `MetricsPage` extend `TreeNavigationPage` despite the fact that is-a rule will be corrupted
1. Make methods in `TreeNavigationPage` static and call them from wrappers in `WebServicesEndpointsPage` to avoid some of the redundancy
1. Keep the state as it is in this pull request
1. Some other?

WDYT?